client/src/components/v2/CameraRecordingStatus.js

```diff
@@ -0,0 +1,93 @@
+import React, { useCallback, useState } from 'react';
+import { useChannel, emit } from 'api/v2/socket';
+import formatBytes from 'utils/formatBytes';
+// import { Card } from 'react-bootstrap';
+import PropTypes from 'prop-types';
+
+/**
+ * @typedef {object} CameraRecordingStatusProps
+ * @property {'primary'|'secondary'} device Camera screen
+ */
+
+/**
+ *  Status of a camera
+ *
+ * @param {CameraRecordingStatusProps} props Props
+ * @returns {React.Component<CameraRecordingStatusProps>} Component
+ */
+export default function CameraRecordingStatus({ device }) {
+  /**
+   * Format a camera recording status payload into JSX
+   *
+   * @param {string} payload Payload structure is defined in https://www.notion.so/V3-MQTT-Topics-66e6715d0e1941ffaa82020e5868fbae. See /v3/camera/recording/status/<primary/>secondary>
+   * @returns {React.Component} Component
+   */
+  function formatPayload(payload) {
+    const data = JSON.parse(payload);
+    if (!data) return null;
+
+    let info;
+    let mins; // only used for status "recording"
+    let secs; // only used for status "recording"
+    switch (data.status) {
+      case 'off':
+        info = null;
+        break;
+
+      case 'recording':
+        mins = Math.floor(data.recordingMinutes);
+        secs = (data.recordingMinutes - mins) * 60;
+        info = (
+          <div>
+            <p>{`File Name: ${data.recordingFile}`}</p>
+            <p>{`Time: ${mins} minutes ${secs} seconds`}</p>
+          </div>
+        );
+        break;
+
+      case 'error':
+        info = <p>{data.error}</p>;
+        break;
+
+      default:
```

Comment:
Good work on adding a default scenario for a switch case. I don't think a lot of people remember to do this..